### ISSUE TITLE
Option to add global HTML attributes to component markup

### ISSUE THREAD

**Issue by certainlyakey** [break-stuff/wc-storybook-helpers#44]
Thanks for the truly greatest web components package of the 2024!

I have a question regarding [global HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes) that can be set on any HTML element including custom ones (eg `data-` attributes, `id`, `dir`, etc.). 

Is it possible to pass them somehow to a component so that they would end up in the generated template (on the docs page)?

My current use cases are, for a `data-` attribute to apply some changes to Shadow DOM styling based on the attribute presence (that attribute would apply to all components, hence a prop would not be a good solution), and for `id` to set an identifier for a popup component to be opened when an outside element is clicked.

**Comment by break-stuff** [comment]
@certainlyakey, thank you for the feedback! I'm glad you are able to take advantage of it. 

That's a good question. I think we should be able to map it to the component using the spread directive. Let me play with it a little.

**Comment by break-stuff** [comment]
@certainlyakey, this should be resolved in release 1.4.0. Let me know if you run into any issues.

**Comment by certainlyakey** [comment]
Thanks, it's working well! For reference, an extra attribute can be added simply by setting its value in `args`: 

```
export const WithForcedDarkMode: Story = {
  ...Default,
  name: 'With Forced Dark Mode (using data attribute)',
  args: {
    'data-color-scheme': 'dark',
  },
};
```

The resulting HTML is: 

```
<my-button data-color-scheme="light" appearance="primary">
    Default content
</my-button>
```
